src/background/ToolBarAction.ts

```diff
@@ -6,14 +6,15 @@ import {
   getTabGroupSetting,
   groupTabsBySetting,
 } from "../data/repository/TabGroupSettingRepository";
+import { setToolbarIconBehavior } from '../data/repository/ToolbarRepository'
```

Comment:
Please use double quotes as per the lint rules.

Reply:
```
npm run check
...
npm run check:fix
```

Got it. I force pushed an updated commit (and github seems to have picked it up). So here's hoping for the best.